### PR TITLE
Add SDXL lightning

### PR DESCRIPTION
**Pull request by philipkiely-baseten**
Tested on my account, works amazing!

### CODE REVIEW

File: stable-diffusion/sdxl-lightning/config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - transformers==4.35.2
+  - diffusers==0.23.1
+  - hf_transfer==0.1.4
+  - xformers==0.0.22
+  - accelerate==0.24.1
+resources:
+  cpu: "3"
+  memory: 20Gi
+  use_gpu: true
+  accelerator: A100:1
```
Comment (squidarth) [inline stable-diffusion/sdxl-lightning/config.yaml:25]: If you leave this as `A100`, `1` is the default.

I'd also consider leaving out the cpu/memory requirements here unless there's something specific that we need.

**Review by squidarth — APPROVED** [review]
lgtm!

File: stable-diffusion/sdxl-lightning/config.yaml (hunk 1 above)
Comment (philipkiely-baseten) [reply stable-diffusion/sdxl-lightning/config.yaml:25]: I can just remove those lines entirely?

**Review by htrivedi99 — APPROVED** [review]
looks great!

**Pull request merged**